controllers/server.go

```diff
@@ -83,17 +83,24 @@ func getStatus(w http.ResponseWriter, r *http.Request) {
 		DB           bool   `json:"db_connected"`
 		Broker       bool   `json:"broker_connected"`
 		LicenseError string `json:"license_error"`
```

Comment:
Can this be a bool? Then you could set it to servcfg.IsPro

Reply:
done